linkcheck/director/console.py

```diff
@@ -90,8 +89,11 @@ def internal_error(out=stderr, etype=None, evalue=None, tb=None):
     print_app_info(out=out)
     print_proxy_info(out=out)
     print_locale_info(out=out)
-    print(os.linesep,
-      _("******** LinkChecker internal error, over and out ********"), file=out)
+    print(
+        os.linesep,
```

Comment:
Out of scope for this PR, but this is bogus code.  As [the docs](https://docs.python.org/3/library/os.html#os.linesep) say

>  Do not use _os.linesep_ as a line terminator when writing files opened in text mode (the default); use a single `'\n' `instead, on all platforms.

I suppose outputting \r\r\n on Windows ends up looking just like if you'd output \r\n, so this is mostly harmless.

Also, this prints a space after the newline, which I'm not sure is wanted?